base/sort.jl

```diff
@@ -1925,7 +1946,8 @@ Characteristics:
   * *stable*: preserves the ordering of elements which compare
     equal (e.g. "a" and "A" in a sort of letters which ignores
     case).
-  * *not in-place* in memory.
+  * *not in-place* in memory — requires a temporary
+    array of half the size of the input array.
```

Comment:
```suggestion
  * *not in-place* in memory.
```
not seeing the removal from sort.md in this PR, and also not a fan of this note because it may be subject to change